### ISSUE TITLE
Adding more details to the feedback rubric in "Teaching is a skill"

### ISSUE THREAD

**Issue by arieldeardorff** [carpentries/instructor-training#1178]
In [Teaching is a skill](https://carpentries.github.io/instructor-training/11-practice-teaching/index.html) we introduce the idea of giving feedback with the 2x2 rubric (content and presentation + -) but the current lesson doesn't really describe what falls under content and what is considered presentation feedback, and learners in our recent workshop struggled with the difference. I think it would be great to have a new visual with the rubric and some examples of feedback in each quadrant, or introduce the [rubric from our teaching demos](https://carpentries.github.io/instructor-training/demos_rubric/) at this point.

**Comment by brownsarahm** [comment]
I like the idea of the demo rubric here.  I'm pretty sure I (and maybe others) implicitly use that rubric when I describe content at this point anyway. Making that explicit should be a good thing for our trainees.

**Comment by arieldeardorff** [comment]
> @arieldeardorff I also found the content piece hard, since they are teaching from Carpentries curriculum so we hope the content is good. It has always felt to me that most of what we talk about is therefore down to delivery. The closest it gets to content is what the individual chooses to cover/not cover given time constraints or whether the actually get something wrong.

Definitely! The only other content feedback I give is around whether they setup the lesson well and situated the learner.

**Comment by brownsarahm** [comment]
We had trainees last week say that the content piece was hard to do before we gave them the rubric.  Also when we gave them the rubric, they were worried that they didn't understand the task because it hadn't changed much.  

In real time, it occurred to me that the way it is makes it simpler the first time by giving them fewer things to look at and building the detail of the rubric over the multiple practice teaching instances.  That might be worth considering, but I think the concreteness is probably worth the complexity.

**Comment by rachida88** [comment]
I agree. Same with me
